### PR TITLE
feature/86cuwqpa7 Added Booking Details

### PR DESCRIPTION
**Pull request by gericW**
localhost:3000/booking-details

### CODE REVIEW

File: behind-the-veil-siteroot/imports/ui/components/pages/request-booking/BookingDetails.jsx
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/request-booking/BookingDetails.jsx:1]: This doesnt have a route to go to it

File: behind-the-veil-siteroot/imports/ui/App.jsx
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/App.jsx:1]: Can you remove the prettier stuff? it makes unnecessary changes to the code

File: behind-the-veil-siteroot/imports/ui/components/pages/request-booking/StatusToolTip.jsx
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/request-booking/StatusToolTip.jsx:1]: Check if the status tool tip exists already in the booking cards

**Review by nethbtju — CHANGES_REQUESTED** [review]
Changed needed

File: behind-the-veil-siteroot/imports/ui/App.jsx
Comment (gericW) [reply behind-the-veil-siteroot/imports/ui/App.jsx:1]: just remove it completely ?

File: behind-the-veil-siteroot/imports/ui/components/pages/request-booking/StatusToolTip.jsx
Comment (gericW) [reply behind-the-veil-siteroot/imports/ui/components/pages/request-booking/StatusToolTip.jsx:1]: ok
